### ISSUE TITLE
Cim deserializing error for MSFT_xSPWorkManagementServiceApp.

### ISSUE THREAD

**Issue by andy-cox** [PowerShell/xSharePoint#200]
I'm receiving an error while trying to compile my configurations using the latest version of xSharePoint (0.11.0.0).

Here's the error message:

```powershell
ImportCimAndScriptKeywordsFromModule : Cim deserializer threw an error when deserializing file C:\Program
Files\WindowsPowerShell\Modules\xSharePoint\0.11.0.0\DSCResources\MSFT_xSPWorkManagementServiceApp\MSFT_xSPWorkMana
At C:\windows\system32\windowspowershell\v1.0\Modules\PSDesiredStateConfiguration\PSDesiredStateConfiguration.psm1:
+ ...     $null = ImportCimAndScriptKeywordsFromModule -Module $mod -Resour ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], PSInvalidOperationException
    + FullyQualifiedErrorId : System.Management.Automation.PSInvalidOperationException,ImportCimAndScriptKeywordsFr

ImportCimAndScriptKeywordsFromModule : Syntax Error. Number value is not valid: .
 At line:30, char:19
 Buffer:
nt;
    [Write] System.^UInt32
At C:\windows\system32\windowspowershell\v1.0\Modules\PSDesiredStateConfiguration\PSDesiredStateConfiguration.psm1:
+ ...     $null = ImportCimAndScriptKeywordsFromModule -Module $mod -Resour ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], CimException
    + FullyQualifiedErrorId : Microsoft.Management.Infrastructure.CimException,ImportCimAndScriptKeywordsFromModule
```

It seems to have something to do with the type definition ```System.UInt32``` in the schema.mof. I'm using a clean install of Windows Server 2012 and the latest WMF 5.0 (24th Feb 2016).

I'm fairly new to Dsc but have had no problems if I downgrade to version 0.9.0.0.

**Comment by BrianFarnhill** [comment]
Hey @andy-cox - this is a duplicate of issue #171 and it has been resolved in version 0.12, which should be released to the PowerShell gallery this week. If you want the fix in the mean time, you can grab the [updated MOF file from the release branch](https://github.com/PowerShell/xSharePoint/blob/release-v0.12/Modules/xSharePoint/DSCResources/MSFT_xSPWorkManagementServiceApp/MSFT_xSPWorkManagementServiceApp.schema.mof). Closing this one off, let me know if you have any further issues with it.

**Comment by andy-cox** [comment]
@BrianFarnhill - apologies! I did search but must have missed that issue. Thanks for the info.

**Comment by BrianFarnhill** [comment]
No sweat - I would rather people report duplicates than not report things at all :)

**Comment by magnustomte** [comment]
@BrianFarnhill - any news when v0.12 will hit PSGallery? It is unfortunate that the available version (v0.11) is a non-functional version. :)

**Comment by BrianFarnhill** [comment]
@magnustomte Yea I'm not sure what the hold up on this one is - it's on my list to chase though. I'll advise as soon as I have something on this. In the mean time you can download from the release branch here if you need to fix sooner rather than later. My apologies for the delay on this though!
